### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by marcus-crane** [marcus-crane/october#43] [Announcements]
This release builds Windows versions with `CGO_ENABLED=1`, fixing #40 so October should now work for Windows users.

I'll try to test out any changes on Windows more often as I personally run on macOS but feel free to submit any issues for anything you spot.

**Full Changelog**: https://github.com/marcus-crane/october/compare/v1.0.1...v1.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/marcus-crane/october/releases/tag/v1.0.2'>v1.0.2</a>.</em>

**Comment by Desertzune** [comment]
When attempting to sync this morning, I received an error: **There was a problem sending your highlights: Received a non-200 status code from Readwise: code 400**. It looks more like a Readwise issue, so I will be sending it to them as well.

**Reply by marcus-crane** [reply]
400 indicates a client-side error whereas a 500 indicates a server error so it's more likely to be an issue with October not being able to read some bookmarks.

If you're able to, it'd be helpful if you can forward your logs to `readwise@utf9k.net` so I can have a look at what went wrong.

You can find the location for logs here: https://github.com/marcus-crane/october#issues
